engine/gamesys/src/gamesys/scripts/script_physics.cpp

```diff
@@ -414,10 +414,8 @@ namespace dmGameSystem
         dmGameObject::HCollection collection = dmGameObject::GetCollection(sender_instance);
         void* world = dmGameObject::GetWorld(collection, context->m_ComponentIndex);
         if (world == 0x0)
```

Comment:
Our code style is to wrap the entire statement within {} (for code style, we usually refer to the existing code, e.g. first the current file, then the other files in the library and so on)
E.g. 

```
if(expr)
{
    statement;
}
```

Reply:
fixed! Thank you